README.md

```diff
@@ -64,6 +65,28 @@ to anything in the `ruff` submodule (which includes all of the Rust source code)
 See the
 [contributing guide](./CONTRIBUTING.md) for more details.
 
+### Show Your Support
+
+If you're using ty, consider adding the ty badge to your project's `README.md`:
+
+```md
+[![ty](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ty/main/assets/badge/v0.json)](https://github.com/astral-sh/ty)
+```
+
+...or `README.rst`:
+
+```rst
+.. image:: https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ty/main/assets/badge/v0.json
```

Comment:
I know we have it in ruff but I'm leaning towards omitting this section here. Mainly because I don't have an expectations that users show their support this way. I think the badge at the top is enough.

Reply:
I think you'd be surprised at the number of people that can't wait to start using ty. But I can also see how it could be cleaner to indeed keep it minimal here (too).